### DISCUSSION TITLE
Discord Server?

### DISCUSSION THREAD

**Discussion by danic85** [makerforgetech/modular-biped#73] [Polls]
All the cool kids seem to be on Discord these days. How do we feel about creating one for the community?

Enough votes and I'll add it to the list ;)

**Comment by WhoIsMrSentry** [comment]
I think you should open a server even if you don't get enough votes.In this way, the community expands and a space is created where we can talk about features more easily.

**Reply by danic85** [reply]
It sounds like you're volunteering to be a moderator ;)

**Reply by WhoIsMrSentry** [reply]
I am nominating my candidacy to become a moderator 🤣
